composeApp/src/commonMain/kotlin/com/swmansion/routinetracker/screen/HomeScreen.kt

```diff
@@ -42,6 +44,21 @@ fun HomeScreen(navController: NavController) {
                         } == true,
                     onClick = { homeNavController.navigate(Routines) },
                 )
+                NavigationBarItem(
+                    icon = {
+                        Icon(
+                            painter = painterResource(Res.drawable.ic_settings),
+                            contentDescription = "Settings",
+                            modifier = Modifier.size(24.dp),
+                        )
+                    },
+                    label = { Text("Settings") },
+                    selected =
+                        currentBackStackEntry?.destination?.hierarchy?.any {
+                            it.hasRoute(Settings::class)
+                        } == true,
```

Comment:
Let’s extract this logic to an extension function so we don’t have to duplicate it.